articles/transfer-learning-with-deep-learning/index.md

```diff
@@ -0,0 +1,121 @@
+
+However, transfer learning takes the contrary approach to the isolated nature of traditional machine learning approaches. The basis of this approach is the transferability of prior knowledge to tackle new tasks. There is no isolation. It is possible to leverage knowledge from pre-trained models to train new models. Here, learning of a new task is dependent on tasks that were learned previously. The result of such is a faster and more efficient learning process.
+
+Now that we understand the distinction between the two approaches, it is important to understand when to use deep transfer learning.
+
+#### When to use Deep Transfer Learning
+
+##### Insufficient training data
+
+For efficient implementation of a deep learning model on a task, a lot of data is required. Yet, it is common to be in scenarios where there is insufficient labeled training data. It is cumbersome to attempt to label data for a deep learning model from scratch. In tasks where domain expertise is required to create a quality dataset, it is quite a time consuming to attempt to gain domain knowledge. Deep transfer learning comes into play here by allowing us to use pre-trained models from a related task, on a new task. We need not worry about training data as a result.
+
+##### Optimization
+
+Transfer learning optimizes the machine learning process. It greatly reduces the training time of a model. This is by allowing the use of a different model with prior knowledge related to a given problem to solve the given problem. In addition to saving time, this also makes the learning process more efficient. For instance, one spends less time worrying about training and more time on how to improve the performance of the model.
+
+##### Positive environmental impact
+
+It is important to consider social good. At a time when it is imperative to reduce the carbon footprint in whichever way possible, large deep learning models harm the environment. Their carbon footprint is massive. The results of a study carried out in this [paper](https://arxiv.org/abs/1906.02243) found that the process of training a large neural network can emit five times more CO2 than the lifetime emissions of an average car.
+
+Taking the above into account, transfer learning reduces the training stage of models which results in the use of computational resources more efficiently.
+
+### Types of Deep Transfer Learning
+
+It’s worth noting that, depending on the literature you may come across, the concepts detailed below are related to, or heavily involved in the concept of transfer learning. Some can also be used independently from transfer learning.
+
+#### Domain Adaptation
+
+Domain adaptation involves a scenario where marginal probabilities of the source domain and target domain differ.
```

Comment:
Yes, so the marginal probabilities P(X) = Summation X in P(X,Y). You can express the definition above as follows: The X is the same, whereas the Y's are varying. I have taken a 2-D joint PDF as an example. You may choose whatever you feel apt.